src/css/style.css

```diff
@@ -0,0 +1,66 @@
+/* General Layout */
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* Container Layout */
+.container {
+  width: 1140px;
+  margin: auto;
+}
+
+/* Header Layout */
+.header {
```

Comment:
Add the background color of white.

Reply:
Noted.

Added to the **BODY** tag.

Reply:
Add it also in the header, the header is fixed and currently, it has a transparent background.

Reply:
Ok.